oshmem/mca/spml/ucx/spml_ucx_component.c

```diff
@@ -491,8 +492,16 @@ static int mca_spml_ucx_component_fini(void)
 
 
     ret = opal_common_ucx_mca_pmix_fence_nb(&fenced);
-    if (OPAL_SUCCESS != ret) {
-        return ret;
+    switch (ret) {
+        case PMIX_SUCCESS:
+            break;
+
+        case PMIX_ERR_UNREACH:
+            /* It's ok if we are disconnected, continue cleanup */
+            fenced = 1;
+            break;
+
+        default: return ret;
```

Comment:
imo it is easier to read
```suggestion
    if (ret == PMIX_ERR_UNREACH) {
        /* It's ok if we are disconnected, continue cleanup */
        fenced = 1;
    } else if (ret != PMIX_SUCCESS) {
        return ret;
    }
```

Reply:
ok, makes sense